pep-0700.rst

```diff
@@ -0,0 +1,110 @@
+PEP: 700
+Title: Additional Fields for the Simple API for Package Indexes
+Author: Paul Moore <p.f.moore@gmail.com>
+PEP-Delegate: Donald Stufft <donald@stufft.io>
+Discussions-To: https://discuss.python.org/t/pep-700-additional-fields-for-the-simple-api-for-package-indexes/20177
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Content-Type: text/x-rst
+Created: 21-Oct-2022
+Post-History: `21-Oct-2022 <https://discuss.python.org/t/pep-700-additional-fields-for-the-simple-api-for-package-indexes/20177>`__
+
+
+Abstract
+========
+
+:pep:`691` defined a JSON form for the "Simple Repository API". This allowed
+clients to more easily query the data that was previously only available in
+HTML, as defined in :pep:`503`.
+
+This proposal adds two *optional* additional fields to the JSON form, which
+allow it to be used in place of PyPI's `JSON API <https://warehouse.pypa.io/api-reference/json.html>`__
+in a number of situations.
+
+- A field to allow retrieval of a list of all the published versions of a project.
+- Fields containing the size and upload time for a project file.
+
+
+Specification
+=============
+
+The proposal adds three new fields to the JSON form of the "project detail"
```

Comment:
The abstract says two fields.

```suggestion
The proposal adds two new fields to the JSON form of the "project detail"
```

Reply:
The abstract's wrong. I fixed one but not the other, and you picked the wrong one to correct 🙂

Fix incoming, thanks for spotting this.